LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 The ECHOES Research Project
```

Comment:
Echoes from the Past: Unveiling a Lost Soundscape with Digital Analysis.

Year is fine.

Reply:
I just updated the project name!

Reply:
Just want to make sure, "open-access and **with no restrictions**" means that it's viewable to the public and everyone can clone/fork to contribute, right? I wouldn't let non-maintainer/collaborator have direct push privilege :/

Reply:
Got it, thanks!